pages/components/Button/button.css

```diff
@@ -0,0 +1,105 @@
+.btn-display-wrapper{
+    margin: 1rem 0;
+    padding:2rem;
+    gap:3rem;
+}
+
+.btn{
+    border: none;
+    background: none;
+    padding: none;
+    font-size: inherit;
+}
+
+.btn-primary{
```

Comment:
suggestion can add `cursor: pointer;` property

Reply:
i forgot to add it ,thank you!